subprojects/platform-play/src/testFixtures/resources/org/gradle/play/integtest/fixtures/external/basicplayapp/build.gradle.old

```diff
@@ -1,5 +0,0 @@
-plugins {
-    id 'org.gradle.playframework' version '0.6'
```

Comment:
We can not remove this and the corresponding integration tests in the `external` package before they were moved to https://github.com/gradle/playframework/.